core/frontend/src/render/webgl/PlanarClassifier.ts

```diff
@@ -467,10 +467,8 @@ export class PlanarClassifier extends RenderPlanarClassifier implements RenderMe
     if (undefined === viewState)
       return;
 
-    // TBD - Refine resolution calculation -- increase height based on viewing angle.
-    // But make sure that we don't make it larger than the hardware supports, keeping in mind that we will be doubling the height later.
```

Comment:
Are you still going to double the height later and if so don't you still need to ensure requiredHeight < maxTextureSize?

Reply:
No - that comment was old - the doubling is no longer required.